OpenSim/Common/TableSource.h

```diff
@@ -202,21 +202,30 @@ class TableSource_ : public Component {
                          TimeOutOfRange, 
                          time, timeCol.front(), timeCol.back());
 
-        const auto colInd = _table.getColumnIndex(columnLabel);
+        const auto colInd = 
+            static_cast<int>(_table.getColumnIndex(columnLabel));
```

Comment:
> I think there are a lot of places where an int is (incorrectly) used as an argument or return type in OpenSim.

I thought a potential rationale for returning an int from `getSize()` was to avoid the possibility of accidentally wrapping through zero when subtracting (which could go unnoticed, e.g., as an intermediate calculation in a long expression)—but I agree with your use of size_t. Anyway, I was just curious; no need to hold up the PR.